contributors/devel/api-conventions.md

```diff
@@ -272,7 +272,7 @@ desired state.
 
 The PUT and POST verbs on objects MUST ignore the "status" values, to avoid
 accidentally overwriting the status in read-modify-write scenarios. A `/status`
-subresource MUST be provided to enable system components to update statuses of
+subresource MUST be provided to enable system components to update statuses and metadata of
```

Comment:
@liggitt I also noticed https://github.com/kubernetes/kubernetes/issues/45539.

If I were to create a new `/status` subresource (this is with respect to CustomResources), do you think it should touch metadata then?

cc @deads2k